src/styles/components/header.scss

```diff
@@ -67,11 +67,91 @@
 	&__menu-icon {
 		display: flex;
 		justify-content: center;
+		cursor: pointer;
 
 		@include responsive('tablet') {
 			display: none;
 		}
 	}
+
+	&__menu-icon-hamburger {
+		pointer-events: none;
+	}
+}
+
+.menu {
```

Comment:
lets move this into its own scss file and change it to `header__menu`

Reply:
Done

Reply:
still needs renaming to `header__menu` or something similar, I think `menu` might be a little too generic